app/src/main/java/org/mifos/mobilebanking/utils/CheckSelfPermissionAndRequest.java

```diff
@@ -141,4 +142,81 @@ public void onClick(DialogInterface dialog, int which) {
             }
         }
     }
+
+
+    @TargetApi(Build.VERSION_CODES.JELLY_BEAN)
+    public static void requestPermissionFragment(final Fragment fragment,
```

Comment:
@therajanmaurya Please check this approach and suggest alternatives/improvements.
I have added this so that I can get the PermissionResult Callback in fragment.